Samples/outlook-check-item-categories/manifest-configurations/unified/manifest.json

```diff
@@ -0,0 +1,211 @@
+{
+    "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.22/MicrosoftTeams.schema.json",
+    "id": "dac64749-60d7-4579-9036-cb7308b6d444",
+    "manifestVersion": "1.22",
+    "version": "1.0.0",
+    "name": {
+        "short": "Check item categories on send",
+        "full": "Office Add-ins Outlook Smart Alerts Sample"
+    },
+    "description": {
+        "short": "Checks mail item categories on send.",
+        "full": "This sample checks a mail item's categories before the item is sent."
+    },
+    "developer": {
+        "name": "Contoso",
+        "websiteUrl": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/outlook-check-item-categories",
+        "privacyUrl": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/outlook-check-item-categories",
+        "termsOfUseUrl": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/outlook-check-item-categories"
+    },
+    "icons": {
+        "outline": "assets/outline.png",
+        "color": "assets/color.png"
+    },
+    "accentColor": "#230201",
+    "localizationInfo": {
+        "defaultLanguageTag": "en-us",
+        "additionalLanguages": []
```

Comment:
additionalLanguages is not a required property, so I don't think it should be present when the array is empty.